pyfr/__main__.py

```diff
@@ -242,7 +243,8 @@ def process_partition_add(args):
         if args.partitioner:
             part = get_partitioner(args.partitioner, pwts, ewts, opts=opts)
         else:
-            for name in sorted(cls.name for cls in subclasses(BasePartitioner)):
+            for name in sorted(cls.name for cls in
```

Comment:
Where possible I try and avoid multi-line for loops.  For example, an alternative might be:
```python
partitioners = sorted(cls.name for cls in subclasses(BasePartitioner))
for name in partitioners:
```
which is the same number of lines but conveys a little bit more information.

Reply:
that is longer than the original version :), I can use `parts` to be consistent with the above line. What is your line length limit set to?

Reply:
Python code is 79 per PEP-8 (that extra 1 catches me out quite often!) with multi-line comments wrapped at 72.  Documentation is 72, too.

Reply:
Another solution can be to pull things further up.  For example moving `partitioners = sorted(cls.name for cls in subclasses(BasePartitioner))` (or a suitably renamed version) to the top of the function before the `with`.